minfraud/help.md

```diff
@@ -340,6 +349,7 @@ Refer to the minFraud API documentation for explicit details.
 
 # Version History
 
+* 2.0.0 - New titles for some output variables
```

Comment:
this change seems like it warrants a minor version bump, as titles are optional fields. They only appear in the UI and do not affect backwards compatibility. I suggest version 1.1.0, or simply leaving it as 1.0.1 because we never did a mass release of plugins after the last hackathon

Reply:
title changes are major version updates.

Reply:
Latest minFraud is https://market.komand.com/plugins/rapid7/minfraud/1.0.0

so we should combine entries, and remove the 1.0.1 listing from the Changelog since it wasn't released

```suggestion
* 2.0.0 - New spec and help.md format for the Hub | New titles for some output variables
```